Assets/Scripts/Source/Game/UI/MenuInput.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+
+
+public class MenuInput : MonoBehaviour {
+    public EventSystem eventSystem;
+    public GameObject lastSelectedObj;
+    public GameObject selectedObject;
+
+    public Color selectedColor;
+    public Color unSelectedColor;
+    private bool buttonSelected;
+    private bool movedMenu;
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (buttonSelected)
+        {
+            //If user clicks off of currently selected object
+            if (eventSystem.currentSelectedGameObject == null && selectedObject != eventSystem.currentSelectedGameObject && Input.GetAxisRaw("Vertical") == 0 && !Input.GetMouseButtonDown(0))
```

Comment:
Nit: could use `!eventSystem.currentSelectedGameObject` instead of `eventSystem.currentSelectedGameObject == null`